samples/Calling/eslint.config.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  baseConfig,
+  baseGlobalIgnores,
+  globalConfig,
+  testFilesBaseConfig
+} from '../../common/config/ESLint/eslint.config';
+import { globalIgnores } from 'eslint/config';
+
+export default globalConfig([
+  baseConfig,
+  testFilesBaseConfig,
+  {
+    rules: {
+      'headers/header-format': 'off',
```

Comment:
Why are we disabling the header-format check for the samples?

Reply:
I think I just need to set up the custom rule I see in the config now ill just port it over.